File: chap4/Portee.py
# Ces programmes sont sous licence CeCILL-B V1.
def v (x):
   global z
   u = x * x
   z = int(x)
   return u

y = 4
t = 1 / y
# En python une variable non initialisée n'existe pas donc la variable globale x n'existe pas !
print( v(t) )
#print(z)

File: chap4/Portee.java
// Ces programmes sont sous licence CeCILL-B V1.
class Portee {

  static int y,z;

  static double v (double x) { 
    double u;
    u = x * x;
    z = (int)x;
    return u;}

  public static void main (String [] args) {
    double t;
    y = 4;
    t = 1 / (double)y;
    System.out.println(v(t));}}
